src/mbgl/gl/types.hpp

```diff
@@ -96,5 +96,11 @@ enum class UniformDataType : uint32_t {
     SamplerCube = 0x8B60,
 };
 
+enum class BufferUsageType : uint32_t {
+    StreamDraw = 0x88E0,
+    StaticDraw = 0x88E4,
+    DynamicDraw = 0x88E8,
+};
```

Comment:
I added them in bd46d60 along with assertions for other values that weren't covered yet. Can you review?